android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlBroadcastReceiver.java

```diff
@@ -178,7 +189,7 @@ public void onListObtained(boolean successful) {
             } else if (intent.getBooleanExtra(TransportConstants.PING_ROUTER_SERVICE_EXTRA, false)) {
                 //We were told to wake up our router services
                 boolean altServiceWake = intent.getBooleanExtra(TransportConstants.BIND_REQUEST_TYPE_ALT_TRANSPORT, false);
-                didStart = wakeUpRouterService(context, false, altServiceWake, device);
+                didStart = wakeUpRouterService(context, false, altServiceWake, device, vehicleType);
```

Comment:
@kostyaBoss On line 192 you can remove the parameter `vehicleType` from the call to `wakeUpRouterService(...)`.

Reply:
@santhanamk, can't, it should receive the vehicleType